tests/utils_tests.py

```diff
@@ -1162,3 +1164,15 @@ def test_get_form_data_token(self):
         assert get_form_data_token({"token": "token_abcdefg1"}) == "token_abcdefg1"
         generated_token = get_form_data_token({})
         assert re.match(r"^token_[a-z0-9]{8}$", generated_token) is not None
+
+    def test_get_database_ids(self) -> None:
+        world_health = db.session.query(Dashboard).filter_by(slug="world_health").one()
+        dash_id = world_health.id
+        database_ids = get_database_ids(dash_id)
+        assert len(database_ids) == 1
+
+        world_slice = (
```

Comment:
it is safe to assume that examples_db is behind the chart, you can do
```
assert database_ids == [get_examples_db().id]
```
instead

Reply:
what is `get_examples_db()`? I didn't find any existed usage for this function.